### DISCUSSION TITLE
Welcome to html-tag-quiz Discussions!

### DISCUSSION THREAD

**Discussion by YuriDevAT** [CorinaMurg/html-tag-quiz#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CorinaMurg** [comment]
Hello Everyone!
My name is Corina. I am a former high school math teacher, now learning about web development. I love thinking about accessibility when building projects and I'm excited about the html quiz! I know we will create a fun and very useful app (and hopefully by the end of the project we will know all the tags by heart)! 😎

**Reply by YuriDevAT** [reply]
Hi Corina 👋🏽 Glad to have you as a contributor! The app makes so much progress since you are contributing.

**Comment by CorinaMurg** [comment]
I would love to suggest a few features for the app. These are definitely more than we might have time to implement, so no pressure!
1. A counter that tells the user how many tags they have entered
2. Maybe a congrats message if they entered a certain number of tags already (let’s say 50, 100 ..) or how many left until a complete list
3. Or, a congrats message if they entered all the tags that start with a certain letter (Congrats, you have now entered all the a-letter tags!)
4. Hints that are revealed when the user clicks a button. (I know, it’s a quiz, so maybe we should not help, but sharing the idea anyway in case it might inspire related features.) For example: total number of tags, number of tags that start with a certain letter, which letters of the alphabet are NOT used to start a tag, etc.

Done! (for now 😊)

**Reply by YuriDevAT** [reply]
These are some great suggestions, most of them I also thought about.

- To 1. what do you think about a progress bar?
- To 2. I thought about implementing Micro Animation like https://lottiefiles.com/ . The animation itself could be on the progress bar to the steps you suggested (starting with 50%)
- To 3. I like the idea. There should also be a larger message/animation when all tags are entered. We should be careful though of implementing messages in an accessible way (using the right tags, attributes, aria and such) Research should be done on how to work with pop up messages.
- To 4. I like the idea of the hint button. It should be a "funny" experience using the app and that everyone can uses it and no only an input field with no further information. For the beginning, I suggest to add a subheading/introduction on how to use the app, like: feel in all html tags (only the letters, no `<>` needed, also the deprecated ones (because they still matter when reading older code).)

**Reply by YuriDevAT** [reply]
Let's move this discussion to the discussion section 👉🏽 https://github.com/YurisCodingClub/html-tag-quiz/discussions/20 (I think you are able to open discussions at any point.) After you copy/paste your input I will add my comment on it as well)

**Comment by okudeazinyi1** [comment]
Hi everyone, my name is Thomas and I just completed data analysis course I want to learn programming. Please feel free to collaborate with me I am ready to learn. Am interested in learning R and Python.

**Reply by YuriDevAT** [reply]
Hi Thomas. Good for you that you are joining the Open Source community. Hopefully, you find a project which offers the languages you are interested in 🚀

**Comment by okudeazinyi1** [comment]
My name is Thomas Ingyefu,
I recently completed a professional course on Data analysis and want to learn programming as a beginner.
